### ISSUE TITLE
[Bug]: A pdf can't be opened on windows

### ISSUE THREAD

**Issue by claws99** [LinwoodDev/Butterfly#417]
### What happened?

right click pdf > open with > Linwood Butterfly

### Version

v2.0.0-beta.1

### Platform

Windows

### Relevant log output

```shell
FormatException: Missing extension byte (at offset 11)
  
  #0      _Utf8Decoder.convertSingle (dart:convert-patch/convert_patch.dart:1747)
  #1      Utf8Decoder.convert (dart:convert/utf.dart:351)
  #2      Utf8Codec.decode (dart:convert/utf.dart:63)
  #3      noteDataMigrator (package:butterfly_api/src/converter/note.dart:13)
  #4      new NoteData.fromData (package:butterfly_api/src/models/data.dart:27)
  #5      _ProjectPageState.load (package:butterfly/views/main.dart:153)
  <asynchronous suspension>
```

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by CodeDoctorDE** [comment]
The error means: This is not a butterfly file.
But I can look why it doesn't detect the file extension pdf as pdf document

**Comment by CodeDoctorDE** [comment]
Fixed in 2.0.0-beta.2 (will be released tomorrow)
